CODEOWNERS

```diff
@@ -40,12 +43,14 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # IE Tests:
 /inference-engine/tests/  @openvinotoolkit/openvino-ie-tests-maintainers
 /inference-engine/tests_deprecated/  @openvinotoolkit/openvino-ie-tests-maintainers
+/inference-engine/tests/functional/inference_engine/ngraph_reader  @GlebKazantaev @ichuraev
+/inference-engine/tests/functional/inference_engine/transformations  @GlebKazantaev @ilyachur
 
 # MO:
 /model-optimizer/  @openvinotoolkit/openvino-mo-maintainers
 
 # nGraph:
-/ngraph/  @openvinotoolkit/openvino-mo-maintainers
+/ngraph/  @GlebKazantaev @ilyachur
```

Comment:
the group is added

Reply:
It was resolved in the PR: https://github.com/openvinotoolkit/openvino/pull/628

Reply:
Agree